datalad/interface/base.py

```diff
@@ -510,6 +510,16 @@ class defining a datalad command
 
     # build standard doc and insert eval_doc
     spec = getattr(cls, '_params_', dict())
+
+    # update defaults for any opts in spec that can override eval_defaults
+    overriders = dict()
+    for opt in spec.keys():
+        if opt in eval_params.keys():
+            overriders[opt] = spec[opt]
+    for opt in overriders.keys():
+        xx = spec.pop(opt)
+    eval_defaults.update(overriders)
```

Comment:
So, it seems like you added `result_renderer` to `Siblings._params_` to get access to it here.  But that's not necessary, because it's already an attribute on the class.  This means you can take care of all the overrides, without adding each override to the class's `_params_`.  Perhaps something like this:

```diff
diff --git a/datalad/interface/base.py b/datalad/interface/base.py
index 9c47d029a..618dc0282 100644
--- a/datalad/interface/base.py
+++ b/datalad/interface/base.py
@@ -511,14 +511,10 @@ class defining a datalad command
     # build standard doc and insert eval_doc
     spec = getattr(cls, '_params_', dict())
 
-    # update defaults for any opts in spec that can override eval_defaults
-    overriders = dict()
-    for opt in spec.keys():
-        if opt in eval_params.keys():
-            overriders[opt] = spec[opt]
-    for opt in overriders.keys():
-        xx = spec.pop(opt)
-    eval_defaults.update(overriders)
+    if hasattr(cls, '_no_eval_results'):
+        add_args = None
+    else:
+        add_args = {k: getattr(cls, k, v) for k, v in eval_defaults.items()}
 
     # get docs for eval_results parameters:
     spec.update(eval_params)
@@ -527,7 +523,7 @@ class defining a datalad command
         cls.__call__, spec,
         prefix=alter_interface_docs_for_api(cls_doc),
         suffix=alter_interface_docs_for_api(call_doc),
-        add_args=eval_defaults if not hasattr(cls, '_no_eval_results') else None
+        add_args=add_args,
     )
 
     # return original
```

Re: the datalad-container failure: I confirmed locally that your change triggers it (and the proposed diff above does not), though I haven't yet debugged it to understand what's going on.